pkgs/development/tools/google-app-engine/python-sdk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs, stdenv, python27Full, python27Packages, makeWrapper }:
+with {
+  python = python27Full;
```

Comment:
do you need all of the extra modules? Would it be possible to instead pick only the exact modules you need?